src/js/controllers/onboarding/collectEmail.js

```diff
@@ -30,14 +30,20 @@ angular.module('copayApp.controllers').controller('collectEmailController', func
     walletId = data.stateParams.walletId;
     wallet = profileService.getWallet(walletId);
     $scope.data.accept = true;
+    walletService.getAddress(wallet,false,function(err,addr){
+      if(!err){coinAddress=addr;}else{$log.debug('Error getting address');}
+    });
   });
 
```

Comment:
This section of code should be moved into the `walletService.getAddress` function